### PR TITLE
Added warning for lack of browser support for the ALLOW-FROM option

### PR DESCRIPTION
**Pull request by eligolding**
See here: https://developer.mozilla.org/en-US/docs/Web/HTTP/X-Frame-Options#Browser_compatibility

### CODE REVIEW

**Comment by EvanHahn** [comment]
Great addition. Could you add a link to that MDN article in your PR?

**Comment by eligolding** [comment]
Done. I also added a link to OWASP's Clickjacking Defense Cheat Sheet that explicitly says that you will have no clickjacking defense if used in unsupported browser.

File: README.md [hunk 1]
```diff
@@ -23,3 +23,5 @@ app.use(frameguard('allow-from', 'http://example.com'));
 ```
 
 **Limitations:** This has pretty good (but not 100%) browser support: IE8+, Opera 10.50+, Safari 4+, Chrome 4.1+, and Firefox 3.6.9+. It only prevents against a certain class of attack, but does so pretty well. It also prevents your site from being framed, which you might want for legitimate reasons.
+
+**Warning:** The `ALLOW-FROM` header option is [not supported in most browsers](https://developer.mozilla.org/en-US/docs/Web/HTTP/X-Frame-Options#Browser_compatibility). Those browsers will ignore the entire header, [and the frame *will* be displayed.](https://www.owasp.org/index.php/Clickjacking_Defense_Cheat_Sheet#Limitations_2)
```
Comment (EvanHahn) [inline README.md]: Tiny nitpick: could you move the period _outside_ of the link? Should look like this:

```markdown
... [and the frame *will* be displayed](https://www.owasp.org/index.php/Clickjacking_Defense_Cheat_Sheet#Limitations_2).
```

**Comment by EvanHahn** [comment]
Other than my tiny nitpick, this is great and ready to go.

**Comment by eligolding** [comment]
Nitpicking is good. I'm still new to this markdown thing. Fixed now.

**Comment by eligolding** [comment]
Thanks!

**Comment by EvanHahn** [comment]
I'm building a list of contributors for a new Helmet.js website. Do you have a name and/or website you want me to use to credit you?

**Comment by eligolding** [comment]
Thanks! I do not have a personal website, so you can just use my name Eli Golding. If you want you can also attribute www.qualitybath.com. This contribution was made on their time.

**Comment by EvanHahn** [comment]
Added you to the list! Stay tuned for the full website.

**Comment by eligolding** [comment]
Thanks! So nice of you to do even for such a small contribution. 
Can you use my github profile for my website? Sorry didn't think of that before. Thanks again!

**Comment by EvanHahn** [comment]
Will do!

**Pull request merged**